mappings/net/minecraft/client/audio/Sound.mapping

```diff
@@ -1 +1,4 @@
 CLASS none/cab net/minecraft/client/audio/Sound
+	METHOD e getWeight ()I
+	METHOD f getCopy ()Lnone/cab;
```

Comment:
Both words are interchangeable so I went with the shorter one. Clone is also used by Java, so it may cause confusion.